src/dom.js

```diff
@@ -0,0 +1,95 @@
+import { background, currentDate } from './helper';
+
+// eslint-disable-next-line func-names
+const dom = (function () {
+  function getElement(id) {
+    return document.getElementById(id);
+  }
+  function render(id, cont) {
+    getElement(id).textContent = cont;
+  }
+  function getValue(id) {
+    return getElement(id).value;
+  }
+  function showAlert() {
+    getElement('alert').style.display = 'block';
+  }
+  function hideAlert() {
+    getElement('alert').style.display = 'none';
+  }
+  return {
+    getElement,
+    render,
+    getValue,
+    showAlert,
+    hideAlert,
+  };
+}());
+
+const city = dom.getElement('city');
+const country = dom.getElement('country');
+const temp = dom.getElement('temp');
+const humidity = dom.getElement('humidity');
+const wind = dom.getElement('wind');
+const description = dom.getElement('description');
+const img = dom.getElement('img');
+const date = dom.getElement('date');
+let metrics = true;
+const convBtn = dom.getElement('convert');
+const currentTemp = dom.getElement('temp');
+const unit = dom.getElement('unit');
+
+async function getData(term = 'canada') {
+  const key = 'b5ed054d1747afb3b139aafeac6bf446';
+  const url = `https://api.openweathermap.org/data/2.5/weather?q=${term}&units=metric&APPID=${key}`;
+  const response = await fetch(url, { mode: 'cors' });
+  let data = '';
+  try {
+    data = await response.json();
+    description.innerText = data.weather[0].description;
+    wind.innerText = data.wind.speed;
+    temp.innerText = (data.main.temp).toFixed(1);
+    city.innerText = data.name;
+    country.innerText = data.sys.country;
+    humidity.innerText = data.main.humidity;
+    unit.innerText = 'C';
+    convBtn.innerHTML = '&#176;F';
+    metrics = true;
+    img.setAttribute('src', `https://openweathermap.org/img/wn/${data.weather[0].icon}@2x.png`);
+    dom.hideAlert();
+    background(data.weather[0].main);
+    date.innerText = currentDate();
+    return true;
+  } catch (err) {
+    dom.showAlert();
+    return false;
+  }
+}
+dom.getElement('search').addEventListener('click', () => {
+  const search = dom.getElement('input').value;
+  getData(search);
+});
+
+function convert() {
```

Comment:
[OPTIONAL]
You are supposed to use ES6 syntax for this project, but since it was not previously mentioned in the first review I'd list it as optional but I highly suggest you convert this and all occurrences to ES6 syntax in following way:
- usage of arrow functions instead of the `function` keyword